flow/aggregates.py

```diff
@@ -549,23 +549,23 @@ def _register_aggregates(self, project):
         self._project = project
 
 
-def get_aggregate_id(jobs):
+def get_aggregate_id(aggregate):
     """Generate hashed id for an aggregate of jobs.
 
     Parameters
     ----------
-    jobs : tuple of :class:`~signac.contrib.job.Job`
-        The signac job handles.
+    aggregate : tuple of :class:`~signac.contrib.job.Job`
+        Aggregate of signac jobs.
 
     Returns
     -------
     str
         The generated aggregate id.
 
     """
-    if len(jobs) == 1:
-        return jobs[0].get_id()  # Return job id as it's already unique
+    if len(aggregate) == 1:
+        return aggregate[0].get_id()  # Return job id as it's already unique
 
-    blob = ",".join(job.get_id() for job in jobs)
+    blob = ",".join(job.get_id() for job in aggregate)
```

Comment:
That is reasonable. In signac, about half of the uses of that variable name are strings instead of bytes. This is the only instance of `blob` in signac-flow, so I will go ahead and change it here.